components/ntp_widget_utils/browser/ntp_widget_utils_region.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <vector>
+
+#include "brave/components/ntp_widget_utils/browser/ntp_widget_utils_region.h"
+
+#include "components/prefs/pref_service.h"
+#include "components/country_codes/country_codes.h"
+
+namespace ntp_widget_utils {
+
+bool IsRegionSupported(PrefService* pref_service,
+    const std::vector<std::string> regions,
```

Comment:
Needs an `&` too, should be: `const std::vector<std::string>& regions`

Reply:
woops, good catch, will fix

Reply:
fixed